modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,39 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features or components in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features or components are not intended for production use.
```

Comment:
IS there a reason you are not using the "Technology Preview" call out here (see image as an example)? I think I have only seen TP feature called out using the snippet but I could be wrong. 
![image](https://github.com/user-attachments/assets/18e1f897-204b-4030-9c87-ddfbf9c8d7ce)

Reply:
Follow up: After looking at the table, I realize it may be a little bit difficult to use the TP feature call out snippet in this case, but wanted to mention just in case you think it could be applied.

Reply:
This TP snippet we usually include where we have actually covered the feature details. Here, it is entered as an RN entry and we have already defined the scope using this[ TP link](https://access.redhat.com/support/offerings/techpreview). The place where we have explained the fearure included this snippet in the beginning. So, no need to include this snippet in RN.